src/app/rpmostree-pkg-builtins.cxx

```diff
@@ -310,3 +312,99 @@ rpmostree_builtin_uninstall (int argc, char **argv, RpmOstreeCommandInvocation *
   return pkg_change (invocation, sysroot_proxy, FALSE, (const char *const *)opt_install,
                      (const char *const *)argv, cancellable, error);
 }
+
+struct cstrless
```

Comment:
Ah neat.  You clearly have some C++ experience.  I struggle with it honestly since I need to keep Go and Rust (and Python, and bash) in my working set.